src/server/index.js

```diff
@@ -1,74 +1,70 @@
-var path = require('path')
-const express = require('express')
-const dotenv = require('dotenv');
-const bodyParser = require('body-parser')
-const axios = require('axios');
-const cors = require('cors');
+var path = require("path");
+const express = require("express");
+const dotenv = require("dotenv");
+const bodyParser = require("body-parser");
+const axios = require("axios");
+const cors = require("cors");
 
 dotenv.config();
 
 const API_KEY = process.env.LICENSE_KEY;
-const MAIN_CLOUD_URL = 'https://api.meaningcloud.com/sentiment-2.1';
+const MAIN_CLOUD_URL = "https://api.meaningcloud.com/sentiment-2.1";
 
-const app = express()
+const app = express();
 
-app.use(cors())
-app.use(bodyParser.json())
-app.use(express.static('dist'))
+app.use(cors());
+app.use(bodyParser.json());
+app.use(express.static("dist"));
 
+app.get("/", function (req, res) {
+  res.sendFile("dist/index.html");
+});
 
-app.get('/', function (req, res) {
-    res.sendFile('dist/index.html')
-})
+app.post("/find-meaning", async (req, res) => {
+  console.log(req.body);
+  if (!req.body.url) {
+    return res.status(400).json({ message: "Error: URL is missing" });
+  }
 
-app.post('/find-meaning', async (req, res) => {
-    console.log(req.body)
-    if (!req.body.url) {
-        return res.status(400).json({ message: 'Error: URL is missing' })
-    }
+  let response = null;
 
-    let response = null;
+  try {
+    response = await sendRequest(req.body.url);
+  } catch (error) {
+    console.error(error);
+    return res.status(500).send(error);
+  }
 
-    try {
-        response = await sendRequest(req.body.url);
-    } catch (error) {
-        console.error(error);
-        return res.status(500).send(error);
-    }
+  console.log(response);
 
+  const result = {
+    agreement: response.data.agreement,
+    subjectivity: response.data.subjectivity,
+    confidence: response.data.confidence,
+    irony: response.data.irony,
+  };
 
-    // console.log(response);
-
-    const result = {
-        agreement: response.data.agreement,
-        subjectivity: response.data.subjectivity,
-        confidence: response.data.confidence,
-        irony: response.data.irony
-    }
-
-    console.log(result);
-
-    res.json(result);
-})
+  console.log(result);
 
+  res.json(result);
+});
 
 const sendRequest = async (url) => {
-    var data = JSON.stringify({ "url": url });
-
-    var config = {
-        method: 'post',
-        url: `${MAIN_CLOUD_URL}?key=${API_KEY}&of=json&lang=en`,
-        headers: {
-            'Content-Type': 'application/json'
-        },
-        data: data
-    };
+  var data = JSON.stringify({ url: url });
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/iahmed93/udacity-nd-evaluate-news/pullRequest?prid=8866191)
```suggestion
  var data = JSON.stringify({ url });
```